.travis.yml

```diff
@@ -20,7 +24,7 @@ before_install:
   - composer self-update
```

Comment:
Sorry, I've removed my fork because I thought it doesn't need changes, if we have to remove this I can send another pull request.